### PR TITLE
Progression

### PR DESCRIPTION
**Pull request by fabi1cazenave**
Here’s the first step : enabling automatic lesson progression. Nothing is stored yet, that’s just to test the UX.

### CODE REVIEW

File: dactylo.html [hunk 1]
```diff
@@ -24,15 +24,15 @@ <h1 class="key_list">·</h1>
...
         <select id="layout">
-          <option selected>ergol</option>
-          <!-- <option>erglace</option> -->
-          <option>bepo</option>
-          <option>azerty</option>
-          <option>lafayette</option>
+          <option value="ergol" selected>Ergo‑L</option>
+          <!-- TODO: Erglace -->
+          <option value="bepo">Bépo</option>
+          <option value@"azerty">AZERTY</option>
```
Comment (Nuclear-Squid) [inline dactylo.html:30]: Ntpicks / questions : Why is `AZERTY` in full caps ? Is the `@` in `value@"` a typo ?

File: js/dactylo.js [hunk 2]
```diff
@@ -121,6 +128,7 @@ window.addEventListener('DOMContentLoaded', () => {
       }
     }
 
+    showQuackStatus();
```
Comment (Nuclear-Squid) [inline js/dactylo.js:142]: This is the best function name I have ever seen.

**Review by Nuclear-Squid — APPROVED** [review]
Looks good to me !

File: dactylo.html (hunk 1 above)
Comment (fabi1cazenave) [reply dactylo.html:30]: AZERTY is in full caps because that’s its real name. The `@` is totally a typo.

Comment (fabi1cazenave) [reply dactylo.html:30]: I agree AZERTY looks horrible in full caps. Reverting.

**Pull request merged**